.github/workflows/build.yml

```diff
@@ -61,10 +61,10 @@ jobs:
     # Create development branch at https://github.com/flathub/org.tuxemon.Tuxemon and
     # update the requirements as described in the Readme. Turn it off again when the requirements in the
     # master branch on flathub are aligned again with the Tuxemon requirements
-    - name: Checkout development branch
```

Comment:
This is still required

Reply:
ok, I'm going to fix it, thank you so much!

Reply:
@Murmele I have enabled the line you told me

Reply:
Sorry, I was not able to select the complete code in `build.yml` just revert all changes in there. Only requirements.txt is required.

And another sorry for the confusion
https://github.com/flathub/org.tuxemon.Tuxemon/pull/3
In this mr you made a pullrequest to the master branch. But it has to be against the development branch. The code lines I linked there was just for you that you see that it is the development branch.

1) merge only the changes in requirements.txt
2) change the destimation branch to "development" in the other mr for the flatpak manifest

Reply:
ok, understood